From 0726a9cb4d39423b51d1dbb696e31012a5e85c78 Mon Sep 17 00:00:00 2001
From: Roman Petrovski <roman.petrovski@gmail.com>
Date: Sat, 11 Jun 2016 03:50:00 +0800
Subject: [PATCH 22/38] init: support usbmisc devices

The patch allows to create usbmisc device nodes like /dev/cdc-wdm0.
---
 init/devices.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init/devices.cpp b/init/devices.cpp
index ce6298a9d..369361fbe 100644
--- a/init/devices.cpp
+++ b/init/devices.cpp
@@ -491,7 +491,7 @@ void DeviceHandler::HandleUevent(const Uevent& uevent) {
                    std::find(subsystems_.cbegin(), subsystems_.cend(), uevent.subsystem);
                subsystem != subsystems_.cend()) {
         devpath = subsystem->ParseDevPath(uevent);
-    } else if (uevent.subsystem == "usb") {
+    } else if (uevent.subsystem == "usb" || uevent.subsystem == "usbmisc") {
         if (!uevent.device_name.empty()) {
             devpath = "/dev/" + uevent.device_name;
         } else {
-- 
2.34.1

